telink: b9x: update b9x dts to use it for matter.

BorysNykytiuk opened the pull request:
- update tlsr9x-common.dtsi files.
    - update tlsr9x.dtsi files.

    Signed-off-by: Borys Nykytiuk <borys.nykytiuk@telink-semi.com>

andriy-bilynskyy commented on boards/riscv/tlsr9528a/tlsr9528a.dts:
```diff
@@ -7,3 +7,41 @@
+&pwm0 {
+	/* On board RGB LEDs */
+	pinctrl-ch0 = <&pwm_ch0_pe7_default>;
+	pinctrl-ch2 = <&pwm_ch1_pd1_default>;
```
I think initially that was a bug...

BorysNykytiuk replied:
My bad, changed

andriy-bilynskyy commented on boards/riscv/tlsr9518adk80d/tlsr9518adk80d.dts:
```diff
@@ -7,3 +7,41 @@
+	pwm_pool {
+		compatible = "pwm-leds";
+		out {
+			pwms = <&pwm0 2 PWM_MSEC(20) PWM_POLARITY_NORMAL>,
```
PWM LED at index 0 it's "indication" (yellow if exists otherwise blue)
PWM LEDs at indexes 1..3 RGB fixture according to LES's color, if blue is used just RG
Looking at schematic current data seems to be wrong...
Also for other boards should be checked.

BorysNykytiuk replied:
inspetc all of them

andriy-bilynskyy commented on samples/boards/tlsr9x/pwm_pool/boards/tlsr9528a.overlay:
```diff
@@ -18,8 +18,8 @@
 	pinctrl-ch0 = <&pwm_ch0_pe7_default>;
-	pinctrl-ch2 = <&pwm_ch1_pd1_default>;
-	pinctrl-ch1 = <&pwm_ch2_pd0_default>;
+	pinctrl-ch1 = <&pwm_ch1_pd1_default>;
```
Since corresponding .dts is already in board .dts do we still need these samples overlays? For me it seems confusing. If we keep board overlays we have possibility to quick check Matter functionality.

BorysNykytiuk replied:
It is duplicate info. So we can delete overlays.

andriy-bilynskyy commented:
Why CI Telink build examples has been cancelled after 15s?

BorysNykytiuk committed:
telink: b9x: update b9x dts to use it for matter.

- update tlsr9x-common.dtsi file.
- update tlsr9x.dtsi file.
- update tl3218x-common.dtsi and tl3218x.dtsi

Signed-off-by: Borys Nykytiuk <borys.nykytiuk@telink-semi.com>
